atomic_reactor/inner.py

```diff
@@ -858,19 +858,10 @@ def build_docker_image(self) -> BuildResult:
                 raise
 
             if self.data.build_result.is_image_available():
-                # OSBS2 TBD
-                self.data.built_image_inspect = imageutil.inspect_built_image()
-                # OSBS2 TBD
-                history = imageutil.get_image_history(self.data.image_id)
-                diff_ids = self.data.built_image_inspect[INSPECT_ROOTFS][INSPECT_ROOTFS_LAYERS]
-
-                # diff_ids is ordered oldest first
-                # history is ordered newest first
-                # We want layer_sizes to be ordered oldest first
-                self.data.layer_sizes = [
-                    {"diff_id": diff_id, "size": layer['Size']}
-                    for (diff_id, layer) in zip(diff_ids, reversed(history))
-                ]
+                self.data.built_image_inspect = imageutil.get_inspect_for_image(self.data.image_id)
+                image_archive_tarball_path = "" # TODO
```

Comment:
@chmeliik what would be good way to handle this ? Just create directory in the specific platform build dir ?

Reply:
I'm thinking the best thing to do might be to remove `built_image_inspect` and `layer_sizes` from WorkflowData entirely. The few plugins that need this info can fetch it using imageutil methods instead of taking it from context data.

If it would be too much trouble to remove at this point, maybe just don't set them here and add `# OSBS2 TBD`s about removing them to the affected plugins